Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Processing/ImageObjectImpl.cpp

```diff
@@ -352,6 +353,63 @@ namespace ImageProcessingAtom
         return (float)avg;
     }
 
+    AZ::Color CImageObject::ComputeLinearAlphaWeightedAverageColor() const
```

Comment:
I see we have `CalculateAverageBrightness` (which seems to be unused, can someone at Amazon give us some insight why it was abandoned or by what it was replaced?) just above doing a very similar process, if we really have to do that calculation, could you try grouping the two? I'd rather not open the ability to run a full pixel exploration twice.